tsl/test/expected/dist_policy.out

```diff
@@ -0,0 +1,161 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+\unset ECHO
+psql:include/remote_exec.sql:5: NOTICE:  schema "test" already exists, skipping
+CREATE OR REPLACE FUNCTION test_drop_chunks(job_id INTEGER)
+RETURNS VOID
+AS :TSL_MODULE_PATHNAME, 'ts_test_auto_drop_chunks'
+LANGUAGE C VOLATILE STRICT;
+-- Add data nodes
+\x on
+SELECT * FROM add_data_node('dist_policy_data_node_1', host => 'localhost',
+                            database => 'dist_policy_data_node_1');
+-[ RECORD 1 ]-----+------------------------
+node_name         | dist_policy_data_node_1
+host              | localhost
+port              | 55432
+database          | dist_policy_data_node_1
+node_created      | t
+database_created  | t
+extension_created | t
+
+SELECT * FROM add_data_node('dist_policy_data_node_2', host => 'localhost',
+                            database => 'dist_policy_data_node_2');
+-[ RECORD 1 ]-----+------------------------
+node_name         | dist_policy_data_node_2
+host              | localhost
+port              | 55432
+database          | dist_policy_data_node_2
+node_created      | t
+database_created  | t
+extension_created | t
+
+SELECT * FROM add_data_node('dist_policy_data_node_3', host => 'localhost',
+                            database => 'dist_policy_data_node_3');
+-[ RECORD 1 ]-----+------------------------
+node_name         | dist_policy_data_node_3
+host              | localhost
+port              | 55432
+database          | dist_policy_data_node_3
+node_created      | t
+database_created  | t
+extension_created | t
+
+\x off
+GRANT USAGE ON FOREIGN SERVER dist_policy_data_node_1, dist_policy_data_node_2, dist_policy_data_node_3 TO PUBLIC;
```

Comment:
It's good time to change role from superuser to a user after this granting, e.g.:
```suggestion
GRANT USAGE ON FOREIGN SERVER dist_policy_data_node_1, dist_policy_data_node_2, dist_policy_data_node_3 TO PUBLIC;
SET ROLE :ROLE_1;
```